### PR TITLE
Modification for Standardization of Multilingual Support

### PR DESCRIPTION
**Pull request by shigeya**
Please find discussion materials in Issue #19

***
<a href="https://pr-preview.s3.amazonaws.com/shigeya/vc-wg-charter/pull/33.html" title="Last updated on Dec 16, 2021, 4:52 PM UTC (d749d35)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-wg-charter/33/07a93a4...shigeya:d749d35.html" title="Last updated on Dec 16, 2021, 4:52 PM UTC (d749d35)">Diff</a>

### CODE REVIEW

**Review by kdenhartog — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -150,6 +150,7 @@ <h2>Scope</h2>
           <li>Data models for credentials, presentations, and proofs</li>
           <li>Registries for the data models</li>
           <li>Algorithms for the expression and verification of proofs that use existing cryptographic primitives</li>
+          <li>Multilingual support for the credential data model</li>
```
Comment (msporny) [inline index.html]:
```suggestion
          <li>Refining multilingual support in data models</li>
```

File: index.html [hunk 2]
```diff
@@ -242,6 +243,7 @@ <h3>
             <li>Storage and Sharing of Verifiable Credentials</li>
             <li>Privacy for Verifiable Credentials</li>
             <li>An API for Verifiable Credential Exchange</li>
+            <li>Multilingual Text Mapping Data Model</li>
```
Comment (msporny) [inline index.html]: Not quite clear what this means. It was clear when @shigeya described it on the call. Perhaps something like:

Extensions for binding multilingual strings to GNU gettext-like localization files.

**Review by msporny — CHANGES_REQUESTED** [review]
Just minor changes requested.

File: index.html (hunk 2 above)
Comment (shigeya) [reply index.html]: Thanks. that's clearer.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2021-12-15-vcwg#section1-4) on 2021-12-15

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 1.4. Standardization of Multilingual Support (issue vc-wg-charter#19)

_See github issue [vc-wg-charter#19](https://github.com/w3c/vc-wg-charter/issues/19)._




**Brent Zundel:** One other issue to cover for this.  

**Shigeya Suzuki:** I posted examples yesterday for this and filed a PR for this and got feedback from Kyle and Manu.  
… my proposal has two schemes to do translation and I think now what I'm trying to do is understand it better.  
… the thing that I was wondering how to express new work items in the charter.  

_See github pull request [vc-wg-charter#33](https://github.com/w3c/vc-wg-charter/pull/33)._




**Shigeya Suzuki:** and I'm not sure if it's good now.  
… If certain citations we may want to have some sort of table like list to translate both the key and value.  
… and I wanted to get further feedback on this.  

**Kyle Den Hartog:** Just to comment on the specifics of whether this is adequate..  
… I think the text added to charter makes it good enough, and things you're discussing are things we can get feedback in WG. Good that you're getting jump start on this, when adding text to spec, that's where we can figure out what makes sense..  

**Brent Zundel:** any other comments, questions, concerns?.  
… Those were the VCWG draft charter issues.  
… we've got about 10 minutes left for the other parts of the agenda.  

</details>

File: index.html [hunk 3]
```diff
@@ -242,6 +243,7 @@ <h3>
             <li>Storage and Sharing of Verifiable Credentials</li>
             <li>Privacy for Verifiable Credentials</li>
             <li>An API for Verifiable Credential Exchange</li>
+            <li>Extensions for binding multilingual strings to GNU gettext-like localization files</li>
```
Comment (iherman) [inline index.html]: Only a minor remark: it may be helpful to turn the "GNU gettext-like localization files" into an active link to a specification.

**Review by iherman — APPROVED** [review]

File: index.html (hunk 3 above)
Comment (shigeya) [reply index.html]: Updated. thanks!

**Review by msporny — APPROVED** [review]

**Pull request merged**